src/collaboration-platform/GitHubCollaborationPlatform.ts

```diff
@@ -26,6 +26,13 @@ export class GitHubCollaborationPlatform implements CollaborationPlatform {
     return null;
   }
 
+  async createOrUpdatePRForSingleCommit(
```

Comment:
Should Stacker call the existing `createOrUpdatePRForCommits` instead of defining a new method? Btw, #29 will change `createOrUpdatePRForCommits`'s function signature as it needs more information.

Reply:
Yeah, removing this.